public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/SimplifiedRouting.tsx

```diff
@@ -0,0 +1,161 @@
+  const styles = useStyles2(getStyles);
+  const contactPointsInAlert = getValues('contactPoints');
+
+  const alertManagerMetaData = useGetAlertManagersMetadata();
+
+  const alertManagerMetaDataPostable = alertManagerMetaData.filter((am) => am.postable);
+
+  // we merge the selected contact points with the alert manager meta data
+  const alertManagersWithSelectedContactPoints = alertManagerMetaDataPostable.map((am) => {
+    const selectedContactPoint = contactPointsInAlert?.find((cp) => cp.alertManager === am.name);
+    return { alertManager: am, selectedContactPoint: selectedContactPoint?.selectedContactPoint };
+  });
+
+  // use reducer to keep this alertManagersWithSelectedContactPoints in the state
+  const [alertManagersWithCPState, dispatch] = useReducer(receiversReducer, alertManagersWithSelectedContactPoints);
+
+  function getContactPointsForForm(alertManagersWithCP: AMContactPoint[]) {
+    return alertManagersWithCP.map((am) => {
+      return { alertManager: am.alertManager.name, selectedContactPoint: am.selectedContactPoint };
+    });
+  }
+
+  // whenever we update the receiversState we have to update the form too
+  useEffect(() => {
+    const contactPointsForForm = getContactPointsForForm(alertManagersWithCPState);
+    setValue('contactPoints', contactPointsForForm, { shouldValidate: false });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [alertManagersWithCPState]);
+
+  //todo: decide what to do when there some alert managers not postable
+  //const shouldShowAM = alertManagerMetaDataPostable.length > 1;
+  const shouldShowAM = true;
+
+  return alertManagersWithCPState.map((alertManagerContactPoint, index) => {
+    return (
+      <div key={index}>
+        <Stack direction="column">
+          {shouldShowAM && (
+            <Stack direction="row" alignItems="center">
+              <div className={styles.firstAlertManagerLine}></div>
+              <div className={styles.alertManagerName}>
+                {' '}
+                Alert manager:
+                <img src={alertManagerContactPoint.alertManager.img} alt="" className={styles.img} />
+                {alertManagerContactPoint.alertManager.name}
+              </div>
+              <div className={styles.secondAlertManagerLine}></div>
+            </Stack>
+          )}
+          <Stack direction="row" gap={1} alignItems="center">
+            <ContactPointSelector
+              selectedReceiver={alertManagerContactPoint.selectedContactPoint}
+              dispatch={dispatch}
+              alertManager={alertManagerContactPoint.alertManager}
+            />
+            <LinkToContactPoints />
+          </Stack>
+        </Stack>
+      </div>
+    );
+  });
+}
+
+function LinkToContactPoints() {
+  const styles = useStyles2(getStyles);
+  const hrefToContactPoints = '/alerting/notifications';
+  return (
+    <div className={styles.contactPointsLinkRow}>
```

Comment:
I don't think we need a wrapper here, we can just put the link in the `Text` component.